plugin/api.go

```diff
@@ -86,6 +86,9 @@ type API interface {
 	// GetTeamByName gets a team by its name.
 	GetTeamByName(name string) (*model.Team, *model.AppError)
 
+	// GetTeamsUnreadForUser gets the unread message and mention counts for each team to which the given user belongs.
```

Comment:
Can you document the minimum version required here?

```suggestion
	// GetTeamsUnreadForUser gets the unread message and mention counts for each team to which the given user belongs.
	//
	// Minimum server version: 5.6
```

Reply:
@chclaus Mind taking a look at this? ^

Reply:
+1 on this

Reply:
Done ;)